candi/control-plane-kubeadm/config.yaml.tpl

```diff
@@ -2,14 +2,18 @@
 RotateKubeletServerCertificate default is true, but CIS becnhmark wants it to be explicitly enabled
 https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 */}}
-{{- $featureGates := list "EndpointSliceTerminatingCondition=true" "TopologyAwareHints=true" "RotateKubeletServerCertificate=true" | join "," }}
+{{- $featureGates := list "TopologyAwareHints=true" "RotateKubeletServerCertificate=true" | join "," }}
+{{- if semverCompare "< 1.23" .clusterConfiguration.kubernetesVersion }}
+    {{- $featureGates = list $featureGates "EphemeralContainers=true" | join "," }}
+{{- end }}
```

Comment:
we cannot have feature gates for 1.23, i think rebase is needed

Reply:
deleted